chart/values.yaml

```diff
@@ -255,6 +262,11 @@ cachedAssets:
   # When cleaning the cached assets directory: maximum number of rows to discard.
   maxCleanedRowsNumber: 10000
 
+cachedAssetsS3:
+  bucket: "hf-datasets-server-statics"
```

Comment:
see comment below (use one bucket for cached assets, and another one for assets?)